tutorial/settings.py

```diff
@@ -7,82 +7,82 @@
 #     https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
```

Comment:
We might not need this file as well.

Reply:
I tried removing it but scrapy then complains that it needs this file